service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/PopulateStandardDirectionsHandler.java

```diff
@@ -9,42 +9,36 @@
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.fpl.events.PopulateStandardDirectionsEvent;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
-import uk.gov.hmcts.reform.fpl.model.HearingBooking;
-import uk.gov.hmcts.reform.fpl.service.HearingBookingService;
+import uk.gov.hmcts.reform.fpl.model.Direction;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
 import uk.gov.hmcts.reform.fpl.service.StandardDirectionsService;
 import uk.gov.hmcts.reform.fpl.service.ccd.CoreCaseDataService;
 
+import java.util.List;
 import java.util.Map;
 
-import static uk.gov.hmcts.reform.fpl.model.Directions.getAssigneeToDirectionMapping;
-
 @Component
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class PopulateStandardDirectionsHandler {
     private final CoreCaseDataService coreCaseDataService;
     private final StandardDirectionsService standardDirectionsService;
     private final ObjectMapper mapper;
-    private final HearingBookingService hearingService;
 
     @Async
     @EventListener
     public void populateStandardDirections(PopulateStandardDirectionsEvent event) {
         CaseDetails caseDetails = event.getCallbackRequest().getCaseDetails();
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
```

Comment:
please use CaseConverter instead of objectMapper for case conversion. Object mapper wont populate case state not case Id